datalake-api/src/main/java/com/sequenceiq/sdx/api/model/SdxUpgradeRequest.java

```diff
@@ -30,6 +30,9 @@ public class SdxUpgradeRequest {
     @ApiModelProperty(ModelDescriptions.SKIP_BACKUP)
     private Boolean skipBackup;
 
+    @ApiModelProperty(ModelDescriptions.SKIP_DATAHUB_VALIDATION)
+    private Boolean skipDataHubValidation;
```

Comment:
please add the new field to the `toString` implementation here and in the other request too

Reply:
Done